### PR TITLE
Uhm 4261

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: mirebalais-modules/openmrs/manifests/init.pp [hunk 1]
```diff
@@ -201,6 +206,33 @@
...
+    /* TODO make this more robust so it doesn't copy over if config zip hasn't changed?  */
+    wget::fetch { 'download-openmrs-configuration':
+      source      => "${config_url}",
+      destination => "/tmp/${config_name}.zip",
+      timeout     => 0,
+      verbose     => false,
+    }
+
+    exec{'install-openmrs-configuration':
+      command => "unzip -o /tmp/${config_name}.zip -d /tmp/configuration && rm -rf /home/${tomcat}/.OpenMRS/configuration && mkdir /home/${tomcat}/.OpenMRS/configuration && mv /tmp/configuration/* /home/${tomcat}/.OpenMRS/configuration",
```
Comment (brandones) [inline mirebalais-modules/openmrs/manifests/init.pp]: Should it `rm -rf /tmp/configuration` first to ensure that the unzipped contents don't get mixed with stale files?

**Comment by brandones** [comment]
One comment/question, otherwise looks good

**Comment by mseaton** [comment]
LGTM, @brandones 's comment makes sense.

File: mirebalais-modules/openmrs/manifests/init.pp (hunk 1 above)
Comment (mogoodrich) [reply mirebalais-modules/openmrs/manifests/init.pp]: Good point, I will update!

**Pull request merged**